### PR TITLE
Bump pywin32 version in setup.py, update in requirements.txt

### PR DESCRIPTION
**Pull request by gitpushdashf**
Fixes: #2902

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 extras_require = {
     # win32 APIs if on Windows (required for npipe support)
-    ':sys_platform == "win32"': 'pywin32==227',
+    ':sys_platform == "win32"': 'pywin32>=227',
```
Comment (PrettyWood) [inline setup.py:19]:
```suggestion
    ':sys_platform == "win32"': 'pywin32>=301',
```
maybe? to avoid https://github.com/advisories/GHSA-hwfp-hg2m-9vr2

Comment (gitpushdashf) [reply setup.py]: Sure. I just updated that.

**Comment by yaleman** [comment]
Could this please be merged? 🙏🏻

**Comment by Nicoretti** [comment]
Related PRs (potential duplicates/overlaps):
* #2925 
* #2903 
* #2857 
* #2955

**Comment by gitpushdashf** [comment]
@StefanScherer would it be possible for you to review this or a related MR? This dependency issue is a potential security vulnerability for some, and a false positive for others. We could really use a Docker 5.0.2 release with these changes.

**Comment by jgiannuzzi** [comment]
@ulyssessouza @aiordache @StefanScherer could you please review this PR? 🙏

This dependency issue is causing a lot of trouble for Windows users. It's a potential [security vulnerability](https://github.com/advisories/GHSA-hwfp-hg2m-9vr2), but also prevents installing `docker-py` on Python 3.10, as support for it was [introduced](https://github.com/mhammond/pywin32/blob/b302/CHANGES.txt#L13-L14) in `pywin32` 302.

**Comment by gitpushdashf** [comment]
Unfortunately, it sounds like Docker won't be addressing this and we may be forced to fork.

#2989

**Comment by thrau** [comment]
I understand docker has different priorities, but this is completely blocking the use of docker-py on Windows with Python 3.10, which FWIW, [LocalStack](https://github.com/localstack/localstack) is relying on.

**Comment by milas** [comment]
Hi! Thanks so much for your PR and apologies for the delay in review. A fix for this has been merged and we're planning to issue a new release containing it soon. For context, changes similar to yours were done in #3004 to address some CI changes in the repo, which blocked merging of your PR as-is, and given the delay on this, we wanted to be respectful of our contributor's time and not require you to rebase + re-review.

**Comment by gitpushdashf** [comment]
Thank you!

**Comment by thrau** [comment]
thank you @milas!! :pray:

**Pull request closed without merging**